src/main/java/de/florianmichael/viafabricplus/injection/mixin/fixes/minecraft/item/MixinHeldItemRenderer.java

```diff
@@ -51,16 +67,15 @@ private void transformLegacyBlockAnimations(AbstractClientPlayerEntity player, f
         }
 
         final Arm arm = hand == Hand.MAIN_HAND ? player.getMainArm() : player.getMainArm().getOpposite();
-
         if (blockHitAnimation) {
             applySwingOffset(matrices, arm, swingProgress);
-            matrices.translate(arm == Arm.RIGHT ? -0.14F : 0.14F, 0.12F, 0.12F);
-        } else {
-            matrices.translate(arm == Arm.RIGHT ? -0.15F : 0.15F, 0.07F, 0.12F);
         }
-        matrices.multiply(RotationAxis.POSITIVE_X.rotationDegrees(-102.25f));
-        matrices.multiply((arm == Arm.RIGHT ? RotationAxis.POSITIVE_Y : RotationAxis.NEGATIVE_Y).rotationDegrees(13.365f));
-        matrices.multiply((arm == Arm.RIGHT ? RotationAxis.POSITIVE_Z : RotationAxis.NEGATIVE_Z).rotationDegrees(78.05f));
+        
+        final int direction = arm == Arm.RIGHT ? 1 : -1;
```

Comment:
Move this field directly below the Arm field and add a space between the direction int and the if (blockHitAnimation) part.